libs/test-macros/Cargo.toml

```diff
@@ -9,4 +9,4 @@ proc-macro = true
 [dependencies]
 proc-macro2 = "1.0.26"
 quote = "1.0.2"
-syn = "1.0.5"
+syn = { version = "1.0.5", features = ["full"] }
```

Comment:
Story behind this: we use types gated behind the `full` feature in this crate but we didn't specify this feature here. We also depended on something elsewhere in the workspace which transitively depended on `syn` with this feature enabled, so we got this feature enabled here via cargo unification, but now that the transitive dependency is gone after updating crates, this crate stopped compiling as we started getting exactly what we were asking for in dependencies.